lib/oppen/print_stack.rb

```diff
@@ -21,6 +21,9 @@ def initialize(offset, break_type)
     # IO element that builds the output.
     attr_reader :buffer
 
+    # Config containing customisation flags
```

Comment:
I think in US english, it's `customization`. Damn, I need to install typos in the CI.